### PR TITLE
Add otpod

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/otpod/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - scikit-learn
+    - statsmodels
+    - decorator
+  run:
+    - python
+    - openturns
+    - matplotlib
+    - scikit-learn
+    - statsmodels
+    - decorator
```
Comment (isuruf) [inline recipes/otpod/meta.yaml]: How about adding the requirements here https://github.com/openturns/otpod#requirements

Comment (isuruf) [inline recipes/otpod/meta.yaml]: I meant the version requirements

Comment (jschueller) [inline recipes/otpod/meta.yaml]: They are quite loose, but ok why not.

**Comment by synapticarbors** [comment]
Are we waiting on anything to merge this PR? If not, I'm happy to do so.

**Pull request merged**